commit e4777c2e40f0592214f25aa0320c4505746dfd07
Author: Katsuhiro Suzuki <katsuhiro@katsuster.net>
Date:   Sun Nov 11 00:18:44 2018 +0900

    ASoC: rockchip: add missing slave_config setting for I2S
    
    [ Upstream commit 16a8ee4c80b45984b6de1f90a49edcf336b7c621 ]
    
    This patch adds missing prepare_sleve_config that is needed for
    setup the DMA slave channel for I2S.
    
    Signed-off-by: Katsuhiro Suzuki <katsuhiro@katsuster.net>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/sound/soc/rockchip/rockchip_pcm.c b/sound/soc/rockchip/rockchip_pcm.c
index f77538319221..7029e0b85f9e 100644
--- a/sound/soc/rockchip/rockchip_pcm.c
+++ b/sound/soc/rockchip/rockchip_pcm.c
@@ -32,6 +32,7 @@ static const struct snd_pcm_hardware snd_rockchip_hardware = {
 
 static const struct snd_dmaengine_pcm_config rk_dmaengine_pcm_config = {
 	.pcm_hardware = &snd_rockchip_hardware,
+	.prepare_slave_config = snd_dmaengine_pcm_prepare_slave_config,
 	.prealloc_buffer_size = 32 * 1024,
 };
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1047_e4777c2e40f0592214f25aa0320c4505746dfd07.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1047_e4777c2e40f0592214f25aa0320c4505746dfd07.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
